src/test/java/io/vertx/core/http/HttpMetricsTestBase.java

```diff
@@ -227,6 +228,7 @@ public void testClientConnectionClosed() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
why is that necessary ?

Reply:
re-enabled & fixed now, see latest commit